src/plugins/automod/filters/BadWords.js

```diff
@@ -0,0 +1,54 @@
+const { remove } = require('confusables');
+const Filter = require('../structs/Filter.js');
+
+function compilePattern(pattern) {
+    return new RegExp(...(typeof(pattern) === 'string' ? [pattern, 'i'] : pattern));
+}
+
+function describeMatchedPatterns(preamble, patterns, content) {
+    let patternsDesc = patterns.filter(re => re.test(content)).map(re => `\t\u2022 \`${re}\``).join('\n');
```

Comment:
Yeah yeah if the pattern contains a backtick then the description's gonna look weird.

Reply:
Doesn't twinkle have a function that will format out/escape backticks? (@Dorumin ?)

Reply:
No need to escape backticks, as we control the input; if we ever need to, dependencies could be added to the AutoMod plugin and you can just require the fmt plugin to use its `.code` helper.

Reply:
I'd rather we deal with this now/soon, rather than waiting for it to come back and bite us, but eh.

Reply:
For posterity: <https://discordapp.com/channels/246075715714416641/476452336282107925/672369870167474176>.